naming/src/test/java/com/alibaba/nacos/naming/core/SubscribeManagerTest.java

```diff
@@ -58,6 +58,25 @@ public void getSubscribersWithFalse() {
         }
     }
 
+    @Test
```

Comment:
I suggest you add more cases to test your search function.

Reply:
ok，i will add PushService test case